l10n_pa_reports/model/account_invoice.py

```diff
@@ -2,28 +2,12 @@
 # Copyright 2016 Vauxoo (https://www.vauxoo.com) <info@vauxoo.com>
 from __future__ import division
 import time
-from openerp import models, api, fields
+from openerp import models, api
 
 
 class AccountInvoice(models.Model):
     _inherit = 'account.invoice'
 
-    l10n_pa_concept = fields.Selection([
-        (1, 'Compras o Adquisiciones de Bienes Muebles'),
-        (2, 'Servicios Básicos'),
-        (3, 'Honorarios y Comisiones por Servicios'),
-        (4, 'Alquileres por Arrendamientos Comerciales'),
-        (5, 'Cargos Bancarios, Intereses y Otros Gastos Financieros'),
-        (6, 'Compras o Servicios del Exterior'),
-        (7, 'Compras o Servicios Consolidados')],
-        string='Supplier Invoice Concept',
-        help='Indicates the Concept of the Supplier Invoice.')
-    l10n_pa_prd_srv = fields.Selection([
```

Comment:
@hbto 
IMHO The best solution is create a migration script.
More info in our guidelines of [version-numbers](https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#version-numbers)